pdf12.js

```diff
@@ -0,0 +1,133 @@
+const isThereVowels = (nameStudent) => {
+    const vowels = nameStudent.match(/[AEIOU]/gi);
```

Comment:
שימי לב לדרישת התרגיל - פונקציה שמחשבת את ממוצע הסטודנט )grades.this )ומגדילה את הממוצע 
בנקודה עבור כל אות ניקוד שמופיעה בשמו AEIOU. 

אצלך זה רק מגדיל ב - 1 ולא כל אחד